modules/i3s/src/lib/parsers/parse-zip/signature.ts

```diff
@@ -0,0 +1,2 @@
+/** Description of zip signature type */
+export type ZipSignature = [number, number, number, number];
```

Comment:
Nit. The type is so simple. The better is just declare a variable with this tuple:
```
const zipSignature: [number, number, number, number] = [1,2,3,4];
```
The separate file is too much for it.